src/App.tsx

```diff
@@ -1,17 +1,107 @@
-import React from 'react';
-import './App.css';
-
+import React, { useState } from 'react';
+import './App.scss';
+import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
 
 const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userSelect, setUserSelect] = useState(0);
+  const [currToddos, setTodos] = useState(preparedTodos);
+  const [click, setClick] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+  };
```

Comment:
Nice job! We can add new todos and it's definetly a win. But there're few thing I guess you chould fix.
Firs of all I'm not realy sure you need differnt function for handling changes. You can make logic with one fucntion for both cases

Reply:
Hi! thank for review, i corrected some mistakes, please look again